vnext/Microsoft.ReactNative/Fabric/SwitchComponentView.h

```diff
@@ -0,0 +1,52 @@
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "ComponentView.h"
+
+#include <Microsoft.ReactNative.Cxx/ReactContext.h>
+#include <UI.Xaml.Controls.h>
+#include <react/renderer/components/rnwcore/Props.h>
+#include "ViewComponentView.h"
+
+#pragma warning(push)
+#pragma warning(disable : 4244 4305)
+//#include <react/renderer/components/view/ViewProps.h>
+#pragma warning(pop)
+
+namespace Microsoft::ReactNative {
+
+struct SwitchComponentView : BaseComponentView {
+  using Super = BaseComponentView;
+  SwitchComponentView(winrt::Microsoft::ReactNative::ReactContext const &reactContext);
+
+  std::vector<facebook::react::ComponentDescriptorProvider> supplementalComponentDescriptorProviders() noexcept
+      override;
+  void mountChildComponentView(const IComponentView &childComponentView, uint32_t index) noexcept override;
+  void unmountChildComponentView(const IComponentView &childComponentView, uint32_t index) noexcept override;
+  void updateProps(facebook::react::Props::Shared const &props, facebook::react::Props::Shared const &oldProps) noexcept
+      override;
+  void updateState(facebook::react::State::Shared const &state, facebook::react::State::Shared const &oldState) noexcept
+      override;
+  void updateLayoutMetrics(
+      facebook::react::LayoutMetrics const &layoutMetrics,
+      facebook::react::LayoutMetrics const &oldLayoutMetrics) noexcept override;
+  void finalizeUpdates(RNComponentViewUpdateMask updateMask) noexcept override;
+  void prepareForRecycle() noexcept override;
+  facebook::react::SharedProps props() noexcept override;
+  void handleCommand(std::string const &commandName, folly::dynamic const &arg) noexcept override;
+
+  const xaml::FrameworkElement Element() const noexcept override;
+
+ private:
+  bool m_needsOnLoadStart{false};
```

Comment:
looks like this is used in the other views too, consider moving it to BaseComponentView?

Reply:
I don't want to collapse that too early.  -- There are going to be components that are not elements.  We likely need another class in the middle for ElementComponentView or something.

Text will need something else for its base, since those will end up being `Run`s not UIElements.

I will take a look at what shared logic to move around once I have text more complete.  Since runs will likely complicate things.